apps/docs/joinTheme.config.js

```diff
@@ -0,0 +1,41 @@
+import { SEO_IMG_DEFAULT } from "@calcom/lib/constants";
+
+const themeConfig = {
+  titleSuffix: " | Join.com",
+  logo: <h4 className="m-0">Join.com</h4>,
+  head: (
+    <>
+      <meta name="msapplication-TileColor" content="#ffffff" />
+      <meta name="theme-color" content="#ffffff" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <meta httpEquiv="Content-Language" content="en" />
+      <meta
+        name="description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="og:title" content="Join.com Documentation" />
```

Comment:
this is not Documnetation

it address to link you removed. Just set Join Calendar